packages/waku/src/router/server.ts

```diff
@@ -1,3 +1,3 @@
 export { unstable_defineRouter, unstable_redirect } from './define-router.js';
-export { createPages } from './create-pages.js';
+export { createPages } from './create-pages/common.js';
```

Comment:
I'm sorry for the implicit naming, but "common.ts" has special usage in Waku. We use it when the file is used both from the server and the client.

Reply:
Re: https://github.com/dai-shi/waku/pull/854#discussion_r1749304119

So, we can't use `common.ts`. Unless you have a better suggestion, we should go back with `create-pages.ts` and `create-pages-utils/*.ts`.